package.json

```diff
@@ -39,7 +39,7 @@
   "license": "MIT",
   "peerDependencies": {
     "react": "^16.0",
-    "react-native": ">=0.57 <=0.62"
+    "react-native": ">=0.60 <=0.63"
```

Comment:
I don't think the `React` to `React-Core` change lets us raise this upper bound to 0.63; I think that would best be done as part of #233 (I've just opened #264 for that).

Reply:
Allright it's back to 0.62